pennylane/transforms/optimization/single_qubit_fusion.py

```diff
@@ -23,15 +23,22 @@
 
 
 @qfunc_transform
-def single_qubit_fusion(tape, tol=1e-8):
+def single_qubit_fusion(tape, atol=1e-8, exclude_gates=None):
     """Quantum function transform to fuse together groups of single-qubit
-    operations into the general single-qubit unitary form (:class:`~.Rot`).
+    operations into a general single-qubit unitary operation (:class:`~.Rot`).
+
+    Fusion is performed only between gates that implement the property
+    ``single_qubit_rot_angles``. Any sequence of one or more single-qubit gates
+    (on the same qubit) with that property defined will be fused into one ``Rot``.
 
     Args:
         qfunc (function): A quantum function.
-        tol (float): A tolerance for which to apply a rotation after fusion.
-            If all the angles of rotation are smaller than this amount, no
-            ``Rot`` gate will be applied.
+        atol (float): An absolute tolerance for which to apply a rotation after
+            fusion. If comparison of all fused angles to 0 via ``allclose`` with
+            this ``atol`` returns True, no ``Rot`` gate will be applied.
```

Comment:
```suggestion
        atol (float): An absolute tolerance for which to apply a rotation after
            fusion. After fusion of gates, if the fused angles :math:`\theta` are such that
            :math:`|theta|\leq \text{atol}`, no rotation gate will be applied.
```
Maybe? It reads a little better to me.

Reply:
Oh yeah that's infinitely better, thanks!